.codeclimate.yml

```diff
@@ -98,7 +98,11 @@ plugins:
     enabled: true
 
     exclude_patterns:
-      - docs/*
```

Comment:
Wouldn't be better to leave this? so we can add new files on docs/ and don't have to update this file every time. Also, why

Reply:
You won't have to. We'll only remove entries from here, not add more. When you add a new file, it will be analyzed because it's not excluded here, it will be linted, issues will be fixed, and good to go!

These exclusions are there because those files don't currently comply to markdown style rules, so they're excluded from the markdownlint check. In this PR I linted one of them so I removed the exclusion (by listing all files except for the one that I linted).